ceph & cinder integration

msambol opened the pull request:
Don't need all eyes on this yet, just the Ceph team.

msambol committed:
ceph & cinder/glance integration

ceph folder in example

Fix cinder.enabled

Add quotes around variable

Generate UUID, cinder conf changes

initial glance integration

pool name in glance-api.conf

add ceph-common as dependency

virsh fix

Remove network value

We want to remove the hardcoded value and we'll change it in
all.yml for each CI.

Change file permissions, add lock_path

cjchang commented on envs/example/defaults.yml:
```diff
@@ -107,8 +107,11 @@ nova:
+
 cinder:
-  enabled: True
+  enabled: {{ ceph.enabled }}
```
Just curious, can you enable cinder without enabling ceph?

msambol replied:
A question I have myself. I think the direction we are going is if the customer doesn't get ceph, they don't get cinder. But need to hear back from Vinit or Niraj.

cjchang replied:
I assume that the cinder is supported today without ceph. If the change for cinder to work only with ceph, there may be some issues with the current deployments unless none enabled teh cinder...

msambol replied:
Agreed. Take a look at discussion here: https://github.com/blueboxgroup/ursula/pull/1140.

msambol commented:
I've moved this here: https://github.com/blueboxgroup/ursula/pull/1140

msambol committed:
Moving code to ceph-compute

msambol committed:
Add ceph-compute

We were doing some tasks in cinder-common that really should be done
in a separate task. Namely, these `virsh` commands should only be run
on compute hosts.

omgjlk committed:
Merge pull request #1200 from msambol/ceph-compute

Add ceph-compute

msambol committed:
move ceph-compute below nova
